src/Documentation/doc/advanced/corporation/miscellany.md

```diff
@@ -0,0 +1,154 @@
+# Miscellany
+
+## Corporation's state
+
+Corporation continuously transitions between 5 states: START → PURCHASE → PRODUCTION → EXPORT → SALE → START. 1 cycle of these transitions takes 10 seconds. If you have enough bonus time, it takes 1 second.
+
+START:
+
+- Division:
+  - Office:
+    - Calculate: energy, morale, total experience, salary, employee production by jobs.
+    - Set employees' jobs: copy data from `employeeNextJobs` to `employeeJobs`.
+  - Material market: update `demand`, `competition`, `marketPrice`.
+  - Product market: decrease `demand` and increase `competition`.
+- Calculate corporation's financial statements: revenue, expenses, profit, dividend, total assets, valuation, share price.
+
+PURCHASE:
+
+- Buy input materials.
+- If we have unlocked "Smart Supply", it's used for calculating optimal quantity of input material units.
+
+PRODUCTION:
+
+- Produce output materials and products.
+- If new product's `developmentProgress` is greater than or equal to 100, it's finished. New finished product's stats are calculated in this state.
+
+EXPORT: export output materials.
+
+SALE:
+
+- Sell output materials and product.
+- If we have unlocked "Market-TA1" / "Market-TA2", it's used for calculating selling price.
+
+Special cases: RP and salary expense are increased in 4 states: PURCHASE, PRODUCTION, EXPORT, SALE.
+
+## Import and export
+
+Importing/exporting material is done in EXPORT state.
+
+EXPORT state is before SALE state. It means you sell the material units remained after being exported.
+
+Export string can use "MAX", "EINV", "IINV", "EPROD" and "IPROD". Read the description in export popup for the meaning of these values.
+
+The optimal export string is `(IPROD+IINV/10)\*(-1)`. For example: export "Chemicals" from Chemical division to Agriculture division:
+
+- Agriculture division needs 100 Chemicals/s and has 700 Chemicals in warehouse.
+  - IPROD = -100 ("Consumption is negative production")
+  - IINV = 700
+- "Export" is expressed by number of units per second, so we want to export:
+
+$$\left(100-\frac{700}{10}\right)\equiv\left(-100+\frac{700}{10}\right)\ast(-1)\equiv\left(IPROD+\frac{IINV}{10}\right)\ast(-1)$$
```

Comment:
Here (and elsewhere as well) this is a slightly odd usage of `\equiv`. I think it makes more sense as regular equals.